.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+on: 
+  pull_request:
+      branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-18.04
+        java:
+          - 8
+          # TODO: Java 11 build doesn't work due to Java7 target.
+        include:
+          - os: ubuntu-18.04
+            java: 8
+            coverage: true
```

Comment:
This variable is unused, but im guessing you'll use it later

Reply:
Right, I had to drop jacoco for now, we can look into adding it back in.   Tests are more more immediate concern than coverage right now.